frontend/src/components/AnnouncementsList/AnnouncementsList.js

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from 'react';
+import { useQuery } from 'react-query';
+import Announcement from '../Announcement/Announcement';
+import PropTypes from 'prop-types';
+
+const AnnouncementsList = ({ boardsIds, announcementsNumber, showBoardNames }) => {
+  const { data, isLoading, isError } = useQuery('boardsList', async () => {
+    return await fetch('/boards').then((response) => response.json());
+  });
+
+  const [announcements, setAnnouncemensts] = useState([]);
+  const [windowSize, setWindowSize] = useState('');
+
+  useEffect(() => {
+    const resizeHandler = () => setWindowSize(window.innerWidth);
+    window.addEventListener('resize', resizeHandler);
+
+    return () => {
+      window.removeEventListener(resizeHandler);
+    };
+  }, []);
+
+  useEffect(() => {
+    if (!!data && !!data.boards) {
+      console.log(data.boards.filter(({ boardId }) => boardsIds.includes(boardId)));
```

Comment:
```suggestion
      console.log(data.boards.filter(({ id }) => boardsIds.includes(id)));
```

Board ma swoje inne ID?
Bo jeśli nie to raczej powinno być `id`, 
a nie `boardId`, aby nie dublować

Reply:
done